web/v1/snowflake/sql-runner/sql/standard/00-setup/01-main/01-stored-procedures.sql

```diff
@@ -0,0 +1,426 @@
+  Wraps the above in a transaction.
+  Inputs:
+  META_TBL:                  the metadata table to commit (datamodel_metadata)
+  META_FROM_TBL:             the table from which to insert to metadata
+  STAGE_NXT:                 corresponds to '{{.stage_next}}'
+  STG_SRC_SCHEMA:            the schema for table as source
+  STG_SRC_TBL:               the source table name (to stage)
+  STG_TRG_SCHEMA:            the schema for table as target
+  STG_TRG_TBL:               the target table name (stage)
+  STG_JOIN_KEY:              the join key for staged tables
+  SKIP_DRV:                  corresponds to '{{.skip_derived}}'
+  DRV_SRC_SCHEMA:            the schema for table as source
+  DRV_SRC_TBL:               the source table name (to derive)
+  DRV_TRG_SCHEMA:            the schema for table as target
+  DRV_TRG_TBL:               the target table name (derive)
+  DRV_JOIN_KEY:              the join key for derived table
+  DRV_PARTITION_KEY:         the partition key for derived table
+  DRV_UPSERT_LOOKBACK FLOAT: corresponds to {{.upsert_lookback_days}}::float
+
+  To drop:
+  DROP PROCEDURE {{.output_schema}}.commit_with_metadata(VARCHAR,VARCHAR,VARCHAR,VARCHAR,
+                                                         VARCHAR,VARCHAR,VARCHAR,VARCHAR,
+                                                         VARCHAR,VARCHAR,VARCHAR,VARCHAR,
+                                                         VARCHAR,VARCHAR,VARCHAR,FLOAT);
+
+*/
+CREATE OR REPLACE PROCEDURE {{.output_schema}}.commit_with_metadata(
+                                                        META_TBL            VARCHAR,
+                                                        META_FROM_TBL       VARCHAR,
+                                                        STAGE_NXT           VARCHAR,
+                                                        STG_SRC_SCHEMA      VARCHAR,
+                                                        STG_SRC_TBL         VARCHAR,
+                                                        STG_TRG_SCHEMA      VARCHAR,
+                                                        STG_TRG_TBL         VARCHAR,
+                                                        STG_JOIN_KEY        VARCHAR,
+                                                        SKIP_DRV            VARCHAR,
+                                                        DRV_SRC_SCHEMA      VARCHAR,
+                                                        DRV_SRC_TBL         VARCHAR,
+                                                        DRV_TRG_SCHEMA      VARCHAR,
+                                                        DRV_TRG_TBL         VARCHAR,
+                                                        DRV_JOIN_KEY        VARCHAR,
+                                                        DRV_PARTITION_KEY   VARCHAR,
+                                                        DRV_UPSERT_LOOKBACK FLOAT)
```

Comment:
I think this would be a lot more readable if we used full names for the input variables.